example_dags/example_snowflake_partial_table_with_append.py

```diff
@@ -111,7 +111,13 @@ def example_snowflake_partial_table_with_append():
     append(
         append_table=filtered_data,
         columns=["sell", "list", "variable", "value"],
-        main_table=Table(table_name="homes_reporting"),
+        main_table=Table(
```

Comment:
@dimberman, I think we need to infer connection details from 1st table and pass it explicitly?